### PR TITLE
Rework manual page

### PR DESCRIPTION
**Pull request by augfab**
* Renamed `gophernicus.1.man` to `gophernicus.1`, the standard name for section 1 manual pages.
* Added synopsis of command.
* Added markup for options, arguments, paths, authors, links.

I used the following references:
* [mdoc](https://mandoc.bsd.lv/man/mdoc.7.html) and [roff](https://mandoc.bsd.lv/man/roff.7.html) manual pages for the syntax
* the BSD and Linux manual pages for style.

`roff` is very not convenient to read.
The manual page can be seen in the terminal with
```
$ man ./gophernicus.1
```
and exported to another format (_e.g._ PDF) with:
```
$ man -Tpdf ./gophernicus.1 > ./gophernicus.1.pdf
```
or with emacs' `nroff-mode` with `C-c C-c`.

### CODE REVIEW

File: gophernicus.1 [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+.Sh STANDARDS
+.Nm
+mostly conforms to
+.Lk https://tools.ietf.org/html/rfc1436 "RFC 1436"
+.Sh AUTHORS
+.An -nosplit
+.An Kim Holviala.
+wrote the original implementation (2009\(en2018).
+.Pp
+.An fosslinux
```
Comment (fosslinux) [inline gophernicus.1:224]: And hb9nks.

File: gophernicus.1 [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+Print debug output in
+.Xr syslog 3
+and
+.Pa /server-status .
+.It Fl v
+Display version information and build date.
+.It Fl b
+Display the BSD license.
+.It Fl \&?
+Display this help.
```
Comment (fosslinux) [inline gophernicus.1:210]: Maybe display the help?

File: gophernicus.1 [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+.Lk https://github.com/gophernicus/gophernicus/issues
+.Pp
+Please report any bug you might experience there as well.
+.Sh COPYRIGHT
+Copyright \(co
+.An Kim Holviala
+2009\(en2018.
+.Pp
+Copyright \(co
+.An Nm gophernicus Ns \& developpers
```
Comment (fosslinux) [inline gophernicus.1:245]: developpers typo

File: gophernicus.1 [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+Disable HTTP response to HTTP GET and POST requests.
+.It Fl d
+Print debug output in
+.Xr syslog 3
+and
+.Pa /server-status .
+.It Fl v
+Display version information and build date.
+.It Fl b
+Display the BSD license.
```
Comment (fosslinux) [inline gophernicus.1:208]: How about display licensing information? User might ask why BSD (if they didn't know Gophernicus was licensed under BSD).

File: gophernicus.1 [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+.Op Fl nH
+.Op Fl d
+.Op Fl b
+.Op Fl \&?
+.Sh DESCRIPTION
+.Nm
+is a gopher server. It serves almost compliant RFC 1436, with small
+changes to make it more modern.
+.Nm
+supports userdirs, executable gophermaps, CGI, virtual hosting (in the limits
```
Comment (fosslinux) [inline gophernicus.1:53]: , and virtual hosting (within the ....).

File: gophernicus.1 [hunk 6]
```diff
@@ -0,0 +1,251 @@
+.Dd April 13, 2020
+.Dt GOPHERNICUS 1
+.Os
+.Sh NAME
+.Nm gophernicus
+.Nd modern, full-featured and secure gopher server
```
Comment (fosslinux) [inline gophernicus.1:6]: A modern...

File: gophernicus.1 (hunk 3 above)
Comment (augfab) [reply gophernicus.1]: Thanks, I always get tripped by this one…

**Comment by augfab** [comment]
Fixed typo in @hb9kns' radio call sign.

**Pull request merged**